apps/pie-docs/src/accessibility/accessibility-resources.md

```diff
@@ -0,0 +1,44 @@
+---
+eleventyNavigation:
+    key: Accessibility resources
+    parent: Accessibility
+    order: 2
+title: Accessibility resources
+description: In this section you’ll find a list of resources about accessibility in design systems and digital products.
+---
+
+## Web accessibility guidelines
+
+The Web Content Accessibility Guidelines (WCAG) are the internationally recognised guidelines for web accessibility. They provide guidance on how to make web content more accessible to people with disabilities.
+
+* [WCAG 2.1 overview](https://www.w3.org/WAI/standards-guidelines/wcag/)
+* [Understanding WCAG 2.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/)
+* [ARIA 1.1: Accessible Rich Internet Applications](https://www.w3.org/TR/wai-aria/)
+* [A11y Project accessibility checklist](https://www.a11yproject.com/checklist/)
+
+---
+
+## Accessibility at JET
+
+To help you get a better picture of how we tackle accessibility at JET, we've compiled a list of internal resources and communities which can provide more information and guidance on multiple accessibility topics.
+
+* [Inclusion, Diversity & Belonging at JET](https://kitchen.just-eat.com/home/inclusion-diversity-belonging){target=“_blank”}
+* [JET #accessibility-guild](https://justeat.slack.com/archives/CLJ26MQ4W){target=“_blank”}
+
+---
+
+## Design systems and accessibility
+
+Ensuring that our design system is accessible is crucial to ensuring that all of JET’s products are accessible. Below are some resources related to design systems and accessibility:
+
+* [Inclusive components](https://inclusive-components.design/)
+* [Accessibility in design systems](https://tetralogical.com/blog/2022/06/24/accessible-design-systems/)
+
+---
+
+## Accessible digital products
+
+In addition to ensuring that PIE is accessible, it is essential to ensure that the digital products that are built using our design system are also accessible. Below are some resources related to building accessible digital products:
+
+* [Web accessibility – Intro, why and who's responsible](https://drive.google.com/file/d/1LzS8LVv8waq3Gv3ezMMBjnBRLfO7dhUv/view){target=“_blank”}
```

Comment:
Worth mentioning this is for JET Employees only?

Reply:
@jordifornellsJET

Reply:
We've update this, thanks @xander-marjoram